code/modules/assembly/signaler.dm

```diff
@@ -145,8 +145,9 @@
 		manual_suicide(suicider)
 		return
 	pulse(TRUE)
-	audible_message("[icon2html(src, hearers(src))] *beep* *beep* *beep*", null, hearing_range)
-	playsound(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
+	if(!src.quiet)
+		audible_message("[icon2html(src, hearers(src))] *beep* *beep* *beep*", null, hearing_range)
+		playsound(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
```

Comment:
```suggestion
	audible_message("[icon2html(src, hearers(src))] *beep* *beep* *beep*", null, hearing_range)
	playsound(get_turf(src), 'sound/machines/triple_beep.ogg', ASSEMBLY_BEEP_VOLUME, TRUE)
```

Reply:
of course if you do the above you would have to change that back too sorry i forgot about that